model.js

```diff
@@ -17,8 +17,11 @@ exports.selectArticleById = (articleId) => {
     return connection
       .query(
         `
-        SELECT * FROM articles
-        WHERE article_id = $1
+        SELECT articles.*, 
+(SELECT COUNT(*) FROM comments 
```

Comment:
a sub-query is not needed here

You should be able to use `JOIN`s to join `comments` to `articles` and use that `COUNT` function in the original query